Integrations/python/deephaven_legacy/TableTools/__init__.py

```diff
@@ -373,6 +373,11 @@ def base64Fingerprint(source):
     return _java_type_.base64Fingerprint(source)
 
 
+@_passThrough
+def booleanCol(name, *data):
+    return _java_type_.booleanCol(name, *data)
```

Comment:
So, this code is actually the correct implementation - the custom ones are for primitives. The missing pydoc is because I didn't follow-up with some .json changes (check fails if the python method booleanCol is missing, but not if the pydoc is "wrong"). Since we are axing legacy shortly, I figure it's better to *not* change hundreds of files (other small doc changes have been accumulating, but nobody has re-run the python generator); and leave as-is?

Reply:
Or you could re-run the generator? This is my (paranoid) pre-commit script:
```
./gradlew :replication-static:replicateAllSafe
./gradlew :replication-reflective:replicateAll
./gradlew spotlessApply
rm -r Integrations/python/deephaven/doc/*
./gradlew :Generators:generateAllPython
./gradlew :Generators:groovyStaticImportGenerator
./gradlew spotlessApply
```